packages/relay/src/lib/eth.ts

```diff
@@ -1108,7 +1108,7 @@ export class EthImpl implements Eth {
    *
    * @param {string } blockHashOrBlockNumber The block hash, block number, or block tag
    * @param {RequestDetails} requestDetails The request details for logging and tracking
-   * @returns {Promise<Receipt[]>} Array of transaction receipts for the block
+   * @returns {Promise<Receipt[] | null>} Array of transaction receipts for the block or null if block not found
```

Comment:
Why the difference between `Receipt` and `ITransactionReceipt` in the return type? (I see it was already like this before the change, but if it's a trivial fix we should do it).